.github/workflows/publish_docker.yml

```diff
@@ -31,9 +36,9 @@ jobs:
         env:
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
           DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
-          IMAGE_NAME: "avaplatform/subnet-evm"
           VM_ID: ${{ inputs.vm_id }}
           PUBLISH: 1
           PLATFORMS: "linux/amd64,linux/arm64"
           AVALANCHE_VERSION: ${{ inputs.avalanche_version }}
+          ALLOW_TAG_LATEST: ${{ github.event_name != 'workflow_dispatch' || inputs.allow_tag_latest }}
```

Comment:
nit Perhaps we could have this as `TAG_LATEST` with the expression:

```
${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) && (github.event_name != 'workflow_dispatch' || inputs.allow_tag_latest) }}
```

And remove the master branch check from the script, since it would be done here?